x-pack/test/functional/fixtures/kbn_archiver/reporting/ecommerce.json

```diff
@@ -786,7 +786,7 @@
     ],
     "title": "Customer Betty"
   },
-  "coreMigrationVersion": "8.1.0",
+  "coreMigrationVersion": "8.0.0",
```

Comment:
@tsullivan is it intentional change? It probably will start the migration process and take a bit more time

Reply:
Was unintentional, but as I glanced back at it, I didn't think it would hurt anything. I'll go ahead and revert.

Since this is an existing line of code, I am guessing this line of code is different in the 8.0 branch, if it is there at all.